src/renderers/testing/ReactTestRendererFiberEntry.js

```diff
@@ -611,6 +605,21 @@ var ReactTestRendererFiber = {
         return TestRenderer.getPublicRootInstance(root);
       },
     };
+
+    Object.defineProperties(entry, {
+      root: {
+        get: function() {
+          if (root === null || root.current.child === null) {
+            throw new Error("Can't access .root on unmounted test renderer");
+          }
+          return wrapFiber(root.current.child);
+        },
+        enumerable: true,
+        configurable: true,
```

Comment:
Pretty much every property we have is `enumerable` and `configurable` since that's the default for assignment and object literal construction. So why should this property be different? The reason configurable is useful is that it allows for patching, polyfilling and backward compatibility fixes. The downside is that it might allow for bad patching, polyfilling etc. :)